docs/release-notes/v11.0.2.md

```diff
@@ -13,7 +13,11 @@ _May 15th, 2023_
 
 #### Incorrect retrieval of a cronjob using Kubernetes v1.25
 
-Has been fixed a bug that prevented the correct retrieve of cronjobs using Kubernetes >= v1.25.
+It has been fixed a bug that prevented the correct retrieve of cronjobs using Kubernetes >= v1.25.
+
+#### Identities role management malfunction
```

Comment:
this was not introduced by v11.0.1, it was only a problem on saas infrastructure